### PR TITLE
modified myInfo.md

### PR DESCRIPTION
**Pull request by Kishanjbethur96**
Kishan's first Pull request

### CODE REVIEW

File: Personel/Kishan/myInfo.md [hunk 1]
```diff
@@ -1 +1 @@
-I am Kishan from Davangere
```
Comment (dayanandviveki) [inline Personel/Kishan/myInfo.md:1]: Trying to Review

**Review by dayanandviveki — COMMENTED** [review]
Trying to Review

**Review by abhi3068 — COMMENTED** [review]
add more info

File: Personel/Kishan/myInfo.md [hunk 2]
```diff
@@ -1 +1 @@
-I am Kishan from Davangere, I am a Developer.
+I am Kishan from Davangere, I am a Developer and I want to be a successful team Lead.
```
Comment (dayanandviveki) [inline Personel/Kishan/myInfo.md:1]: Trying to Review

**Review by dayanandviveki — COMMENTED** [review]
We Are Trying to Review Code

**Pull request merged**